24life/blocks/article-hero-video/article-hero-video.js

```diff
@@ -66,21 +70,28 @@ export default function decorate(block) {
   section.append(`${sectionName}`);
 
   const title = document.createElement('h1');
-  title.classList.add('article-hero-video-title');
+  // if (!getMetaInfo) {
+  //   title.classList.add('article-hero-video-title-newsletter');
+  // }
+  // title.classList.add('article-hero-video-title');
+  // eslint-disable-next-line no-unused-expressions
+  (!getMetaInfo) ? title.classList.add('article-hero-video-title-newsletter') : title.classList.add('article-hero-video-title');
   title.innerText = data?.title.trim();
 
   const authorLinks = document.createElement('p');
   authorLinks.classList.add('authors');
-  authorLinks.append('By ');
-  getMetadata('authors').trim().split(',').forEach((author, index) => {
-    const authorLink = document.createElement('a');
-    authorLink.href = `${window.hlx.codeBasePath}/author/${toClassName(author)}`;
-    authorLink.textContent = author;
-    if (index > 0) {
-      authorLinks.append(' and ');
-    }
-    authorLinks.append(authorLink);
-  });
+  if (getMetaInfo) {
+    authorLinks.append('By ');
+    getMetadata('authors').trim().split(',').forEach((author, index) => {
+      const authorLink = document.createElement('a');
+      authorLink.href = `${window.hlx.codeBasePath}/author/${toClassName(author)}`;
+      authorLink.textContent = author;
+      if (index > 0) {
+        authorLinks.append(' and ');
+      }
+      authorLinks.append(authorLink);
+    });
+  }
 
   titleContainer.append(section);
   titleContainer.append(title);
```

Comment:
Instead I would just make an `if` around `titleContainer.append(authorLinks);` in case it's empty.